### PR TITLE
Updated docker image

### LINKED ISSUE #385: Add CA certificates to Docker container
The tegola Dockerfile needs CA certificates in order to make calls to SSL enabled endpoints (i.e. fetching a remote config). The following command needs to be added [here](https://github.com/go-spatial/tegola/blob/master/docker/Dockerfile#L52). 

`RUN apk update && apk add ca-certificates && rm -rf /var/cache/apk/*`

### PR DESCRIPTION
**Pull request by stvno**
working docker for the 0.8.0 branch, should fix #385

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -26,14 +26,16 @@
 
 
 # --- Build the binary
-FROM golang:1.9.2-alpine3.7 AS build
+FROM golang:1.10.3-alpine3.7 AS build
```
Comment (ARolek) [inline docker/Dockerfile]: Can we bump to Go 1.11 now that it's out?

File: docker/Dockerfile [hunk 2]
```diff
@@ -45,12 +47,14 @@ RUN ln -s /opt/tegola_src /opt/tegola_src/src/github.com/go-spatial/tegola
 # Build binary
 WORKDIR /opt/tegola_src/cmd/tegola
 ENV GOPATH=/opt/tegola_src
+RUN go get -u github.com/theckman/goconstraint/go1.8/gte
```
Comment (ARolek) [inline docker/Dockerfile]: Ideally we should vendor this rather than add it here.

File: docker/Dockerfile (hunk 1 above)
Comment (ARolek) [reply docker/Dockerfile]: looks like `golang:1.11.0-alpine3.8` is now available

File: docker/Dockerfile (hunk 2 above)
Comment (ARolek) [reply docker/Dockerfile]: I looked into it and this package is [vendored](https://github.com/go-spatial/tegola/blob/master/vendor/vendor.json#L407). Is the Docker build failing without this line?

Comment (stvno) [reply docker/Dockerfile]: Yes it states:
```
Step 12/19 : RUN go build -v -gcflags "-N -l" -o /opt/tegola
 ---> Running in 662cda042194
main.go:7:2: cannot find package "github.com/theckman/goconstraint/go1.8/gte" in any of:
        /usr/local/go/src/github.com/theckman/goconstraint/go1.8/gte (from $GOROOT)
        /opt/tegola_src/src/github.com/theckman/goconstraint/go1.8/gte (from $GOPATH)
```

File: docker/Dockerfile (hunk 1 above)
Comment (stvno) [reply docker/Dockerfile]: done

File: docker/Dockerfile (hunk 2 above)
Comment (ARolek) [reply docker/Dockerfile]: Strange. Can you check your local copy. I see this in the vendor directory [here](https://github.com/go-spatial/tegola/tree/v0.8.x/vendor/github.com/theckman/goconstraint/go1.8/gte) and I would prefer to keep this go get request out of the Docker image.

**Comment by ARolek** [comment]
Looking into this a bit more I think it has to do with the way the Docker image is being built. The GOPATH is being overwritten and the DockerFile is kind of working against the root image. Let me do a bit more research.

**Comment by gdey** [comment]
#535 Redid this commit.

**Pull request closed without merging**